### PR TITLE
Add config spec

### PR DESCRIPTION
**Pull request by AlexandreYang**
Use config spec.

This PR only change wording and add missing attributes to `conf.yaml.example`

cc @brentm5

### CODE REVIEW

**Review by florimondmanca — APPROVED** [review]
:+1:

Let's also double-check the base requirement is sufficient in `setup.py`, must be at least `datadog-checks-base>=11.0.0`.

**Review by florimondmanca — CHANGES_REQUESTED** [review]
Update: looks like the base requirement should indeed be updated

https://github.com/DataDog/integrations-extras/blob/06d4039d299e41db776ab60de8afcb95c34f3776/resin/setup.py#L27

**Comment by AlexandreYang** [comment]
@florimondmanca What change makes the change not compatible with older base dep ?

**Comment by AlexandreYang** [comment]
@florimondmanca thx, updated the base req https://github.com/DataDog/integrations-extras/pull/736/commits/81f388856fbdf3ed8bb46f54996acb36d26fbf64

File: resin/assets/configuration/spec.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    - template: instances/jmx
+      overrides:
+        host.value.example: localhost
+        port.value.example: 7199
+  - template: logs
+    example:
+    - type: file
+      path: /var/opt/resin/log/*.log
+      source: resin
+      service: <SERVICE_NAME>
```
Comment (ChristineTChen) [inline resin/assets/configuration/spec.yaml:19]:
```suggestion
```

**Review by ChristineTChen — CHANGES_REQUESTED** [review]
nit

**Review by ChristineTChen — APPROVED** [review]
thanks! LGTM

**Pull request merged**